src/vss_tools/vspec/vssexporters/vss2samm/vss2samm.py

```diff
@@ -0,0 +1,298 @@
+
+    global ttl_helper
+    ttl_helper = importlib.import_module("vss_tools.vspec.vssexporters.vss2samm.helpers.ttl_helper")
+
+
+# TODO: Currently this is a workaround to read the Vehicle.VersionVSS, which is provided from COVESA/VSS
+#       and provides the supported VSS version.
+#       In best case this functionality should be provided by the vspec tool, loaded in this script.
+#
+#       Once we migrate the vss2samm tool under the COVESA/vss-tools,
+#       we could move this functionality under the vspec script
+#       and make sure that we read the version from the COVESA/vehicle_signal_specification/VERSION
+#       or as helper function under the VSSNode - root tree.
+def __get_version_vss(vss_tree: VSSNode) -> str:
+    # DEFAULT version would be 1.0.0
+    major = 1
+    minor = 0
+    patch = 0
+
+    if vss_tree.children and len(vss_tree.children) > 0:
+        # Get the VersionVSS node so to extract current VSS version from it.
+        vss_version_node = next(filter(lambda vs_child: vs_child.name == "VersionVSS", vss_tree.children), None)
+
+        if vss_version_node:
+            for v_child in vss_version_node.children:
+                if (
+                    v_child.name in ["Major", "Minor", "Patch"]
+                    and hasattr(v_child.data, "default")
+                    and type(v_child.data.default) is int
+                    and v_child.data.default > 1
+                ):
+                    match v_child.name:
+                        case "Major":
+                            major = v_child.data.default
+                        case "Minor":
+                            minor = v_child.data.default
+                        case "Patch":
+                            patch = v_child.data.default
+
+    return f"{major}.{minor}.{patch}"
+
+
+# Exporter specific options
+@clo.option(
+    "-sigf",
+    "--signals-file",
+    type=click.Path(dir_okay=False, readable=True, path_type=Path, exists=True),
+    help="""\b
+Path to file with selected VSS signals to be converted.
+Allows to convert just selected VSS signals into aspect model(s), if '-spl / --split' is enabled.
+\033[36mNOTE:\033[0m each signal in the file should be on a new line and in the format of:
+      \033[96mPARENT_SIGNAL.PATH.TO.CHILD_SIGNAL\033[0m as defined in VSS.
+\033[33mEXAMPLE:\033[0m \033[96m-sigf PATH_TO_FILE/selected_signals.txt\033[0m
+ """,
+)
+@clo.option(
+    "--split-depth",
+    "-spld",
+    type=int,
+    default=1,
+    show_default=False,
+    help="""\b
+Number - used to define, up to which level, VSS branches will be converted into single aspect models.
+Can be used in addition to the \033[32m-spl, --split\033[0m option.
+Default value of 1 means that only 1st level VSS branches like Vehicle.Cabin, Vehicle.Chassis etc.,
+will be converted to separate aspect models.
+\033[30m[default: 1]\033[0m
+ """,
+)
+@clo.option(
+    "--split/--no-split",
+    "-spl",
+    default=False,
+    show_default=False,
```

Comment:
Should we possibly have "True" here  (and for other args) here to show what is the default.

Reply:
Done.

I actually, defaulted it to True, before to see your comment :)

This is also reflected in the related samm.md doc.